### PR TITLE
circleci: add explicit circleci v2 config

### PR DESCRIPTION
**Pull request by jonboulle**
previously we were just using the default (inferred) circle v1 config, which will stop working soon.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/blinkist/blinkist-config
+    parallelism: 1
+    docker:
+    - image: circleci/ruby:2.3.6-node-browsers
```
Comment (tprzedmojski) [inline .circleci/config.yml]: maybe let's go ruby 2.5.something?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    steps:
+    - checkout
+    - run: 
+        name: Bundle Install
```
Comment (tprzedmojski) [inline .circleci/config.yml]: https://github.com/blinkist/blinkist-core/blob/master/.circleci/config.yml#L20 before this guy?

**Review by tprzedmojski — APPROVED** [review]
Also: please remove the old file

**Review by tprzedmojski — APPROVED** [review]
nice 👍

**Comment by jonboulle** [comment]
ty

**Pull request merged**